src/ImageSharp/Formats/Png/Zlib/Adler32.cs

```diff
@@ -63,7 +63,7 @@ public static uint Calculate(uint adler, ReadOnlySpan<byte> buffer)
             }
 
 #if SUPPORTS_RUNTIME_INTRINSICS
-            if (Sse3.IsSupported && buffer.Length >= MinBufferSize)
+            if (Ssse3.IsSupported && buffer.Length >= MinBufferSize)
             {
                 return CalculateSse(adler, buffer);
```

Comment:
Should we also rename `CalculateSse` to `CalculateSsse`?

Reply:
I would say no. It’s still an iteration of SSE technology.